go/badges/badgestate.go

```diff
@@ -48,9 +48,20 @@ type BadgeState struct {
 }
 
 // NewBadgeState creates a new empty BadgeState.
-func NewBadgeState(g *libkb.GlobalContext) *BadgeState {
+func NewBadgeState(log logger.Logger, env *libkb.Env) *BadgeState {
```

Comment:
why changes here?

Reply:
I needed to update protocols to get the sig3 chaintype, and that spiraled out into me having to update all of client. This function is used by both server and client and was recently changed to use `g`, so after I vendored I had to change it back so server could call it without `g`.